src/app/components/VisualImage.js

```diff
@@ -0,0 +1,26 @@
+function VisualImage(imageSource) {
+  // Checking that img source is in base64 format
+  // FIXME: If we should use only base64/jpg, delete one condition in if 
+  const reg = /^data:image\/(?:gif|png|jpeg|bmp|webp)(?:;charset=utf-8)?;base64,(?:[A-Za-z0-9]|[+/])+={0,2}/;
+  if(reg.test(imageSource) || /.jpg/.test(imageSource)) {
+  // Creating div and img
+  const createDiv = document.createElement("div");
+  createDiv.className = "visual__box";
```

Comment:
Look for [BEM](http://getbem.com/naming/) naming convention. `visual__box` would be ok if owner of this would have a parent with `visual` class. In that case it should be `visual-box`. And if it div would have any child, that child should have class like `visual-box__something`. And if you would want sometimes modify it in some way, for example hide, then class for this should look like this `visual-box__something--hidden`.